README.md

```diff
@@ -110,7 +110,8 @@ With golang, it's generally easiest to have your fork match the import paths in
 rm -rf ${GOPATH}/src/github.com/pachyderm/pachyderm
 mkdir -p ${GOPATH}/src/github.com/pachyderm
 cd ${GOPATH}/src/github.com/pachyderm
-git clone https://github.com/alice/pachyderm.git
+git clone ${YOUR_FORK} pachyderm
```

Comment:
`YOUR_FORK` isn't something implicitly defined anywhere heh, I think the former was less confusing